packages/office-ui-fabric-react/gulpfile.js

```diff
@@ -20,7 +20,7 @@ let {
 } = buildConfig;
 
 let visualTestClean = build.subTask('visualTestClean', (gulp, options, done) => {
```

Comment:
hmm this one seems unnecessary. There is a clean config where we can provide the match. We shouldn't need to tdo this.